src/app/data-planes/components/StandardDataplanePolicies.vue

```diff
@@ -73,7 +65,7 @@
 
             <RuleEntryList
               class="mt-2"
-              :rules="rs"
+              :rules="rs!"
```

Comment:
**Question**: How come this is needed now? Shouldn’t this change be a simple replacement of "groupBy" code that has no other impact?

Reply:
Looks like the Typescript definition can return `[] | undefined` instead of `[]` for a 'group' which as far as I understand it can never happen. The key just isn't there if there is no group and therefore doesn't get looped over. Actually without looking back at it properly I'm not sure if this is the Typescript `groupBy` definition or Vue's for loop.

Reply:
FYI I dug around to get some clarity on this and it seems appropriate (although somewhat annoying) that the result of `Object.groupBy` is a `Partial<...>`, see https://github.com/microsoft/TypeScript/issues/47171#issuecomment-1848451195